classroom.rb

```diff
@@ -0,0 +1,19 @@
+class Classroom
+  attr_accessor :label, :students
```

Comment:
- [ ] [OPTIONAL] I would suggest you to make `students` object read only instead of creating both read and write accessors for the sake of immutability. This is so that program utilising the class, cannot accidentally modify the student data and introduce any bugs.